include/wtlgo/Logger.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef Logger_hpp
+#define Logger_hpp
+
+#include <iostream>
+#include <curl/curl.h>
+
+namespace wtlgo {
+
+class ILogger {
+public:
+	virtual ILogger();
+	virtual ~ILogger();
+	virtual void log(CURLcode log);
```

Comment:
Please, turn this into pure virtual too
```suggestion
	virtual void log(CURLcode log) = 0;
```